packages/desktop-client/src/components/reports/reports/CalendarCard.tsx

```
@@ -0,0 +1,522 @@
+        firstDayOfWeekIdx={firstDayOfWeekIdx}
+        onDayClick={date => {
+          navigate(
+            isDashboardsFeatureEnabled
+              ? `/reports/calendar/${widgetId}?day=${format(date, 'yyyy-MM-dd')}`
+              : '/reports/calendar',
+          );
+        }}
+      />
+      <View style={{ fontWeight: 'bold', fontSize: '12px' }}>
+        <span
+          ref={rel => (monthFormatSizeContainers.current[0] = rel)}
```

Biome

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)